dodola/cli.py

```diff
@@ -136,16 +136,14 @@ def rechunk(x, variable, chunk, maxmemory, out):
     required=True,
     help="Regridding method - 'bilinear' or 'conservative'",
 )
-@click.option(
-    "--targetresolution", "-r", default=1.0, help="Global-grid resolution to regrid to"
-)
+@click.option("--domain_file", "-domain", help="Domain file to regrid to")
```

Comment:
For consistency, on this line I suggest changing

* `--domain_file` to `--domain-file` (click will still pass this to the function as the `domain_file` kwarg).
* `-domain` to `-d`

See https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html.